src/constructs/numbers.md

```diff
@@ -2,6 +2,120 @@
 title: Numbers
 ---
 
-## On the Subject of Numbers
+Grain has a unified number type, called `Number`, which contains integer (whole) numbers, floating-point (decimal) numbers, and rational (fractional) numbers. Grain also has separate integer and float types for programs which require more precise typechecking.
 
-This documentation hasn't yet been migrated here (or written, to be honest). In the meanwhile, please check out the [Grain compiler README](https://github.com/grain-lang/grain/blob/master/README.md) for more information. Consider contributing by fleshing out this page yourself!
\ No newline at end of file
+```grain
+42 // an integer
+```
+
+```grain
+1.23 // a floating-point number
+```
+
+```grain
+1/3 // a rational number
+```
+
+## Number Syntax
+
+### Integers
+
+To make an integer more readable, underscores can be used throughout the number as a visual separator. The underscores do not affect the value of the number.
+
+```grain
+let oneBillion = 1_000_000_000
+```
+
+In addition to decimal numbers, integers can be written in hexadecimal, octal, and binary:
```

Comment:
"In addition to decimal numbers"??

Reply:
Decimal numbers, like base 10. Does it read better if we just drop the word "numbers" here?

Reply:
Oh. I'm a dummy and don't think of "base 10" as "decimal" 🙃 I usually think of "decimal" like "floating point". Wikipedia says it as "Demical notation" - thoughts? I could also see just saying "base-ten representation" or something.

Reply:
I like "decimal notation" so let's go with that 👍 
```suggestion
In addition to decimal notation, integers can be written in hexadecimal, octal, and binary:
```